### ISSUE TITLE
TimeRange constructor fails with array constructor

### ISSUE THREAD

**Issue by Eric-Arellano** [esnet/pond#97]
Hello, I'm having issues getting Pond to work (trying to use `react-timeseries-charts`). One of the bugs I've found is that this code

```javascript
const range = new TimeRange([1326309060000, 1329941520000])
console.log(range.toString())
```
will throw the bug
```javascript
.../node_modules/pondjs/lib/timerange.js:127
        return this._range.get(0);
TypeError: Cannot read property 'get' of undefined
    at TimeRange.begin (.../node_modules/pondjs/lib/timerange.js:127:28)
    at TimeRange.toJSON (.../node_modules/pondjs/lib/timerange.js:80:35)
    at TimeRange.toString (.../node_modules/pondjs/lib/timerange.js:88:36)
```

This is using version `1.0.0-alpha.1` (the default for npm install) and node 9.3.

**Comment by Eric-Arellano** [comment]
This error doesn't occur on the 0.8.7 version. Perhaps consider restoring the npm default to be 0.8.7, especially considering the react charts library doesn't support 1.0 yet?

**Comment by Eric-Arellano** [comment]
Resolved with PR #98
